classes/use-imx-seco-firmware.bbclass

```diff
@@ -0,0 +1,32 @@
+#
+# Class to provide a possibility to re-use names of SECO Firmware
```

Comment:
Suggest to expand the acronym at the beginning to SEcurity COntroller.

Reply:
Good point, would expand it.